jdk/make/CreateJars.gmk

```diff
@@ -590,7 +555,20 @@ $(eval $(call SetupArchive,BUILD_TOOLS_JAR, , \
         META-INF/services/com.sun.jdi.connect.spi.TransportService \
         META-INF/services/com.sun.tools.attach.spi.AttachProvider \
         META-INF/services/com.sun.tools.internal.ws.wscompile.Plugin \
-        META-INF/services/com.sun.tools.internal.xjc.Plugin, \
+        META-INF/services/com.sun.tools.internal.xjc.Plugin \
+        com/ibm/jvm/Debug.class \
```

Comment:
Where do these classes come from? I don't find them.

Reply:
Which classes are you referring to? If you mean the META-INF/services/.... ones, then these were in that make file already and I presumed they are related to OpenJDK8. If you mean the com/ibm/jvm/... ones then these were listed in the icl-build.mk file that I was pointed to which was previously used to build the tracefomat.jar. These classes do not exist as separate java files (which caused me some issues initially) but they are classes that are defined in other java classes (which I find very strange!!!).

Reply:
Ok I see, they are defined in the com.ibm.jvm.TraceFormat.java file.